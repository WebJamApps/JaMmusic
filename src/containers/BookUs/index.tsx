import { Button } from '@mui/material';
import { Inquiry } from '../NewHomepage/Inquiry';

const songlistUrl = 'https://docs.google.com/spreadsheets/d/1uePCuHO6qxg2tHTqGist_O4iYXJjy9AgSmXvMgF9NPM/edit?usp=sharing';
export function BookUs() {
  return (
    <div className="bookus" style={{ margin: 'auto', padding: '6px', paddingRight: '12px' }}>
      <div style={{ maxWidth: '800px' }}>
        <h3 style={{ textAlign: 'center' }}>Book Us</h3>
        <h4 style={{ fontWeight: 'normal' }}>
          We are always looking for regional performance opportunites,
          and would consider traveling even further away from our home in Salem, VA for just the right situation.
          <Button
            style={{ marginLeft: '8px' }}
            size="small"
            variant="contained"
            onClick={() => window.open(songlistUrl)}
          >
            Current Songlist
          </Button>

        </h4>
        <p style={{ textAlign: 'center' }}>
          Please fill out this form and we will get back with you soon.
        </p>
        <Inquiry country="United States" hideTitle />
      </div>
    </div>
  );
}
