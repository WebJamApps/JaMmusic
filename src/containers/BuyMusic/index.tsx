import { Component } from 'react';
import Jsb from './BuyLinks/joshShermanBand';
import Jss from './BuyLinks/joshShermanSolo';
import Jsyt from './MediaLinks/joshShermanYoutube';
import Jssp from './MediaLinks/joshShermanSpotify';
import Jsam from './MediaLinks/joshShermanApple';
import commonUtils from '../../lib/utils';

export default class BuyMusic extends Component {
  commonUtils: { setTitleAndScroll: (pageTitle: string, width: number) => void };

  constructor(props: Record<string, unknown>) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount(): void { this.commonUtils.setTitleAndScroll('Buy Music', window.screen.width); }

  render(): JSX.Element {
    return (
      <div
        className="page-content"
        style={{
          paddingRight: '15px', maxWidth: '800px', margin: 'auto', borderRight: 'none',
        }}
      >
        <h2 style={{
          textAlign: 'center', margin: '20px', fontWeight: 'bold', marginBottom: '10px',
        }}
        >
          Buy from CD Baby
        </h2>
        <div className="row top-row-style">
          <Jsb />
          <Jss />
        </div>
        <Jssp />
        <Jsyt />
        <Jsam />
      </div>
    );
  }
}
