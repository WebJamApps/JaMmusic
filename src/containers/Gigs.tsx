import React, { useContext, useState } from 'react';
import { GigsContext } from '../providers/Gigs.provider';

export const Gigs = (): JSX.Element => {
  const { gigs } = useContext(GigsContext);
  console.log(gigs);
  const [allGigs, setGigs] = useState(gigs);
  return (
    <div style={{ margin: 'auto' }}>
      <h3>Gigs</h3>
      {JSON.stringify(allGigs)}
    </div>
  );
};
