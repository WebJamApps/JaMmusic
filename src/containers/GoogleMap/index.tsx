
import { Component } from 'react';
import UMap from './UserMap';
import CMap from './CompanyMap';
import gMapUtils from './gMapUtils';

export interface Iloc { lat: number, lng: number }
export const equals = () => true;
class GoogleMap extends Component {
  gMap: google.maps.Map | null;

  companyMap: CMap | null;

  userMap: UMap | null;

  content: string;

  gMapUtils: typeof gMapUtils;

  constructor(props: Record<string, unknown>) {
    super(props);
    this.gMap = null;
    this.userMap = null;
    this.companyMap = null;
    this.content = '';
    this.gMapUtils = gMapUtils;
  }

  componentDidMount(): void {
    const mapDiv = document.getElementById('googleMap');
    if (mapDiv) this.gMap = new google.maps.Map(mapDiv, { zoom: 2, center: { lat: 40, lng: -100 } });
    this.userMap = new UMap();
    this.userMap = this.gMapUtils.limitUserLat(this.userMap);
    this.content = `<div><p><strong>User Name:</strong> ${this.userMap.name}</p>`
      + `<p><strong>Latitude:</strong> ${this.userMap.loc.lat}</p></div>`;
    this.addMarker(this.userMap);
    this.companyMap = new CMap();
    this.companyMap = this.gMapUtils.limitCompanyLat(this.companyMap);
    this.content = `<div><p><strong>Company Name:</strong> ${this.companyMap.name}</p>`
      + `<p><strong>Company Slogan:</strong> ${this.companyMap.catchPhrase}</p>`
      + `<p><strong>Latitude:</strong> ${this.companyMap.loc.lat}</p></div>`;
    this.addMarker(this.companyMap, {
      scaledSize: { width: 40, height: 40, equals },
      url: 'https://image.flaticon.com/icons/png/512/63/63838.png',
    });
  }

  addMarker(obj: { loc: Iloc }, icon?: google.maps.Icon | undefined): void {
    const infoWindow = new google.maps.InfoWindow({ content: this.content });
    if (this.gMap !== null) {
      const marker = new google.maps.Marker({
        map: this.gMap,
        position: {
          lat: obj.loc.lat, lng: obj.loc.lng,
        },
        icon,
      });
      marker.addListener('click', () => { // eslint-disable-next-line security/detect-non-literal-fs-filename
        /* istanbul ignore else */if (this.gMap !== null)infoWindow.open(this.gMap, marker);
      });
    }
  }

  render(): JSX.Element {
    return (
      <div className="page-content google">
        <div style={{ margin: 'auto', textAlign: 'center' }}>
          <h3 style={{ marginTop: '10px' }}>Google Map Here</h3>
          <div id="googleMap" style={{ height: '8.5in', maxWidth: '11in', margin: 'auto' }} />
        </div>
      </div>
    );
  }
}
export default GoogleMap;
