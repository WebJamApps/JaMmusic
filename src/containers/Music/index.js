import React, { Component, useEffect } from 'react';
import PropTypes from 'prop-types';
import { connect } from 'react-redux';
import PicSlider from '../../components/pic-slider';
import DefaultTable from '../../components/tour-table';
import JoshBio from './joshBio';
import MariaBio from './mariaBio';
import EmersonBio from './emersonBio';
import BrianBio from './brianBio';
import Wjband from './wjBand';
import Intro from './intro';
 import mapStoreToProps from '../../redux/mapStoreToProps';
 // import MusicDashboard from '../MusicDashboard/index'; 

export class Music extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  componentDidMount() { document.title = 'Music | Web Jam LLC'; }

  musicians() { // eslint-disable-line class-methods-use-this
    return (
      <div className="elevation3" style={{ maxWidth: '1000px', margin: 'auto' }}>
        <section>
          <JoshBio />
          <hr />
          <MariaBio />
          <hr />
          <Wjband />
          <hr />
          <EmersonBio />
          <hr />
          <BrianBio />
        </section>
        <p>{' '}</p>
        <p>{' '}</p>
      </div>
    );
  }

  render() {
    const { images } = this.props;
    return (
      <div className="page-content">
        <div style={{ paddingTop: '1px', paddingBottom: 0, marginBottom: 0 }}>
          <div id="musicSlide1">
            {images.length > 0 ? (<PicSlider data={images} />) : null}
          </div>
          <Intro />
        </div>
        <div
          className="search-table-outer"
          style={{
            position: 'relative', overflowX: 'auto', maxWidth: '96%', margin: 'auto',
          }}
        >
        {/* <DefaultTable /> */}
        </div>
        <div style={{ height: '10px' }}>
          <p>{' '}</p>
        </div>
        {this.musicians()}
        {/* <MusicDashboard /> */}
      </div>
    );
  }
}

Music.defaultProps = { images: [] };
Music.propTypes = {
  images: PropTypes.arrayOf(PropTypes.shape({
    url: PropTypes.string,
    title: PropTypes.string,
  })),
};

export default connect(mapStoreToProps)(Music);
