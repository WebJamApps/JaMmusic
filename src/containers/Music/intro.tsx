import React from 'react';
import { Link } from 'react-router-dom';

const Intro = () => (
  <div
    className="intro"
    style={{
      margin: 'auto', maxWidth: '900px', padding: '6px', paddingBottom: 0,
    }}
  >
    <h4 style={{
      textAlign: 'center', border: 'solid', margin: 'auto', width: '124px', borderWidth: 'thin', backgroundColor: '#d8ecf3',
    }}
    >
      <strong><Link to="/music/songs">Click To Listen</Link></strong>
    </h4>
    <p style={{ marginTop: '10px', marginBottom: '6px' }}>
      Josh and Maria have been performing their music together for over eight years now!
      Whether it is at church, charity events, public venues, or outdoor festivals, this couple will blow your socks off.
    </p>
  </div>
);

export default Intro;
