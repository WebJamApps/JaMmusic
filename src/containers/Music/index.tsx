import { useEffect, useState, useContext } from 'react';
import { AuthContext, Iauth } from 'src/providers/Auth.provider';
import { PicSlider } from 'src/containers/Music/Pictures/PicSlider';
import commonUtils from 'src/lib/utils';
import { Gigs } from './Gigs';
import { Intro } from './intro';
import JoshBio from './joshBio';
import MariaBio from './mariaBio';
import { CreatePicDialog } from './Pictures/CreatePicDialog';
import { EditPicTable } from './Pictures/EditPicTable';

export function Musicians(): JSX.Element {
  return (
    <div className="elevation3" style={{ maxWidth: '1000px', margin: 'auto' }}>
      <section>
        <JoshBio />
        <hr />
        <MariaBio />
      </section>
      <p>{' '}</p>
      <p>{' '}</p>
    </div>
  );
}

interface IphotosSectionProps {
  showEditPicTable: boolean, setShowEditPicTable: (arg0: boolean) => void, isAdmin: boolean,
  setShowCreatePic: (arg0: boolean) => void, showCreatePic: boolean,
}
export function PhotosSection(props: IphotosSectionProps) {
  const {
    showEditPicTable, setShowEditPicTable, isAdmin, setShowCreatePic, showCreatePic,
  } = props;
  return (
    <div style={{ paddingTop: '1px', paddingBottom: 0, marginBottom: 0 }}>
      <div id="musicSlide1">
        {!showEditPicTable ? <PicSlider /> : <EditPicTable setShowTable={setShowEditPicTable} />}
      </div>
      {showEditPicTable ? null
        : <Intro isAdmin={isAdmin} setShowEditPic={setShowEditPicTable} setShowCreatePic={setShowCreatePic} />}
      <CreatePicDialog showDialog={showCreatePic} setShowDialog={setShowCreatePic} />
    </div>
  );
}

export function checkIsAdmin(auth: Iauth, setIsAdmin: (arg0: boolean) => void) {
  let isAdmin = false;
  if (auth && auth.isAuthenticated && process.env.userRoles) {
    const { user: { userType } } = auth;
    const rolesJSON = JSON.parse(process.env.userRoles);
    const { roles } = rolesJSON;
    if (userType && roles.includes(userType)) isAdmin = true;
  }
  setIsAdmin(isAdmin);
}

export function Music(): JSX.Element {
  const { auth } = useContext(AuthContext);
  const [isAdmin, setIsAdmin] = useState(false);
  const [showCreatePic, setShowCreatePic] = useState(false);
  const [showEditPicTable, setShowEditPicTable] = useState(false);
  useEffect(() => commonUtils.setTitleAndScroll('Music', window.screen.width), []);
  useEffect(() => {
    checkIsAdmin(auth, setIsAdmin);
  }, [auth]);
  return (
    <div className="page-content music">
      <PhotosSection
        showEditPicTable={showEditPicTable}
        setShowEditPicTable={setShowEditPicTable}
        isAdmin={isAdmin}
        setShowCreatePic={setShowCreatePic}
        showCreatePic={showCreatePic}
      />
      <div
        className="search-table-outer"
        style={{
          position: 'relative', overflowX: 'auto', maxWidth: '96%', margin: 'auto', zIndex: 0,
        }}
      >
        {showEditPicTable ? null : <Gigs isAdmin={isAdmin} />}
      </div>
      <div style={{ height: '10px' }}>
        <p>{' '}</p>
      </div>
      {showEditPicTable ? null : <Musicians />}
    </div>
  );
}
