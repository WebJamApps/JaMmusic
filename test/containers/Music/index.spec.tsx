/* eslint-disable @typescript-eslint/no-explicit-any */
import renderer from 'react-test-renderer';
import { Music, checkIsAdmin, PhotosSection } from 'src/containers/Music';
import { ClickToListen } from 'src/containers/Music/intro';
import { BrowserRouter } from 'react-router-dom';

describe('/music', () => {
  it('renders the component', () => {
    const music = renderer.create(<BrowserRouter><Music /></BrowserRouter>).toJSON();
    expect(music).toMatchSnapshot();
  });
  it('renders ClickToListen and handles click when joshandmariamusic.com', () => {
    window.open = jest.fn();
    const ctl = renderer.create(
      <BrowserRouter>
        <ClickToListen
          appName="joshandmariamusic.com"
          isAdmin={false}
          setShowCreatePic={jest.fn()}
          setShowEditPic={jest.fn()}
        />
      </BrowserRouter>,
    ).root;
    ctl.findByProps({ variant: 'contained' }).props.onClick();
    // eslint-disable-next-line security/detect-non-literal-fs-filename
    expect(window.open).toHaveBeenCalled();
  });
  it('renders ClickToListen and handles click when web-jam.com', () => {
    window.open = jest.fn();
    const ctl = renderer.create(
      <BrowserRouter>
        <ClickToListen appName="web-jam.com" isAdmin={false} setShowCreatePic={jest.fn()} setShowEditPic={jest.fn()} />
      </BrowserRouter>,
    ).root;
    ctl.findByProps({ variant: 'contained' }).props.onClick();
    // eslint-disable-next-line security/detect-non-literal-fs-filename
    expect(window.open).not.toHaveBeenCalled();
  });
  it('renders ClickToListen and handles click when undefined appName', () => {
    window.open = jest.fn();
    const ctl = renderer.create(
      <BrowserRouter>
        <ClickToListen isAdmin={false} setShowCreatePic={jest.fn()} setShowEditPic={jest.fn()} />
      </BrowserRouter>,
    )
      .root;
    ctl.findByProps({ variant: 'contained' }).props.onClick();
    // eslint-disable-next-line security/detect-non-literal-fs-filename
    expect(window.open).not.toHaveBeenCalled();
  });
  it('checkIsAdmin when true', () => {
    const setIsAdmin = jest.fn();
    const userRoles = process.env.userRoles || '{}';
    const { roles } = JSON.parse(userRoles);
    const auth: any = { isAuthenticated: true, user: { userType: roles[0] } };
    checkIsAdmin(auth, setIsAdmin);
    expect(setIsAdmin).toHaveBeenCalledWith(true);
  });
  it('renders PhotosSection when showEditPicTable is true', () => {
    const props = {
      showEditPicTable: true,
      setShowEditPicTable: jest.fn(),
      isAdmin: true,
      setShowCreatePic: jest.fn(),
      showCreatePic: false,
    };
    const photosSection:any = renderer.create(<PhotosSection {...props} />).toJSON();
    expect(photosSection.children[0].children[0].props.className).toBe('editPicTable');
  });
});
