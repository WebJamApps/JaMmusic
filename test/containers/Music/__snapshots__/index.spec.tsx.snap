// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/music renders the component 1`] = `
<div
  className="page-content music"
>
  <div
    style={
      {
        "marginBottom": 0,
        "paddingBottom": 0,
        "paddingTop": "1px",
      }
    }
  >
    <div
      id="musicSlide1"
    />
    <div
      className="intro"
      style={
        {
          "margin": "auto",
          "maxWidth": "900px",
          "padding": "6px",
          "paddingBottom": 0,
        }
      }
    >
      <div
        style={
          {
            "margin": "auto",
            "textAlign": "center",
          }
        }
      >
        <button
          className="clickToListen"
          onClick={[Function]}
          variant="contained"
        >
          Click To Listen
        </button>
      </div>
      <h5
        style={
          {
            "fontWeight": "normal",
            "marginBottom": "6px",
            "marginTop": "10px",
          }
        }
      >
        <strong>
          Josh and Maria
        </strong>
         
        have been performing their music together for over 11 years now! Whether it is at church, charity events, public venues, or outdoor festivals, this couple will blow your socks off.
      </h5>
    </div>
    <div
      className="createNewPicDialog"
      disableAutoFocus={true}
      disableEnforceFocus={true}
      onClose={[Function]}
      open={false}
    >
      <div>
        Create New Picture
      </div>
      <div
        sx={
          {
            "padding": "10px 10px",
          }
        }
      >
        <div
          sx={
            {
              "marginBottom": "30px",
            }
          }
        >
          Enter all *required fields to create a new picture.
        </div>
        <input
          fullWidth={true}
          label="* URL"
          onChange={[Function]}
          sx={
            {
              "marginTop": "20px",
            }
          }
          type="text"
          value=""
        />
        <input
          fullWidth={true}
          label="* Title"
          onChange={[Function]}
          sx={
            {
              "marginTop": "20px",
            }
          }
          type="text"
          value=""
        />
        <div>
          <div
            control={
              <Checkbox
                checked={false}
                onClick={[Function]}
              />
            }
            label="Show Title In Caption"
          />
        </div>
      </div>
      <div>
        <button
          className="createPicButton"
          onClick={[Function]}
          size="small"
          variant="contained"
        >
          Create
        </button>
        <button
          className="cancelPicButton"
          onClick={[Function]}
          size="small"
        >
          Cancel
        </button>
      </div>
    </div>
  </div>
  <div
    className="search-table-outer"
    style={
      {
        "margin": "auto",
        "maxWidth": "96%",
        "overflowX": "auto",
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <div
      className="gigsDiv"
      style={
        {
          "margin": "auto",
          "padding": "10px",
          "width": "100%",
        }
      }
    >
      <h4
        style={
          {
            "textAlign": "center",
          }
        }
      >
        Gigs
        <button
          className="bookUsButton"
          onClick={[Function]}
          variant="contained"
        >
          Book Us
        </button>
      </h4>
      <div
        style={
          {
            "height": "500px",
            "width": "100%",
          }
        }
      >
        <table />
      </div>
      <div
        className="createNewGigDialog"
        disableAutoFocus={true}
        disableEnforceFocus={true}
        onClose={[Function]}
        open={false}
      >
        <div>
          Create New Gig
        </div>
        <div
          sx={
            {
              "padding": "10px 10px",
            }
          }
        >
          <div
            sx={
              {
                "marginBottom": "30px",
              }
            }
          >
            Enter all *required fields to create a new gig.
          </div>
          <div
            dateAdapter={[Function]}
          >
            <input
              className="createDatetime"
              label="* Date and Time"
              onChange={[Function]}
              renderInput={[Function]}
              value={2017-01-01T00:00:00.000Z}
            />
          </div>
          <p
            className="venueLabel"
          >
            * Venue
          </p>
          <textarea
            id="create-venue"
            style={
              {
                "visibility": "hidden",
              }
            }
          />
          <input
            fullWidth={true}
            label="* City"
            onChange={[Function]}
            sx={
              {
                "marginTop": "20px",
              }
            }
            type="text"
            value=""
          />
          <div
            fullWidth={true}
            sx={
              {
                "marginTop": "20px",
              }
            }
          >
            <p
              id="create-us-state-label"
            >
              * State
            </p>
            <select
              id="create-us-state"
              label="* State"
              labelId="create-us-state-label"
              onChange={[Function]}
              value="Virginia"
            >
              <option
                value="Alabama"
              >
                Alabama
              </option>
              <option
                value="Alaska"
              >
                Alaska
              </option>
              <option
                value="American Samoa"
              >
                American Samoa
              </option>
              <option
                value="Arizona"
              >
                Arizona
              </option>
              <option
                value="Arkansas"
              >
                Arkansas
              </option>
              <option
                value="California"
              >
                California
              </option>
              <option
                value="Colorado"
              >
                Colorado
              </option>
              <option
                value="Connecticut"
              >
                Connecticut
              </option>
              <option
                value="Delaware"
              >
                Delaware
              </option>
              <option
                value="District of Columbia"
              >
                District of Columbia
              </option>
              <option
                value="Federated States of Micronesia"
              >
                Federated States of Micronesia
              </option>
              <option
                value="Florida"
              >
                Florida
              </option>
              <option
                value="Georgia"
              >
                Georgia
              </option>
              <option
                value="Guam"
              >
                Guam
              </option>
              <option
                value="Hawaii"
              >
                Hawaii
              </option>
              <option
                value="Idaho"
              >
                Idaho
              </option>
              <option
                value="Illinois"
              >
                Illinois
              </option>
              <option
                value="Indiana"
              >
                Indiana
              </option>
              <option
                value="Iowa"
              >
                Iowa
              </option>
              <option
                value="Kansas"
              >
                Kansas
              </option>
              <option
                value="Kentucky"
              >
                Kentucky
              </option>
              <option
                value="Louisiana"
              >
                Louisiana
              </option>
              <option
                value="Maine"
              >
                Maine
              </option>
              <option
                value="Marshall Islands"
              >
                Marshall Islands
              </option>
              <option
                value="Maryland"
              >
                Maryland
              </option>
              <option
                value="Massachusetts"
              >
                Massachusetts
              </option>
              <option
                value="Michigan"
              >
                Michigan
              </option>
              <option
                value="Minnesota"
              >
                Minnesota
              </option>
              <option
                value="Mississippi"
              >
                Mississippi
              </option>
              <option
                value="Missouri"
              >
                Missouri
              </option>
              <option
                value="Montana"
              >
                Montana
              </option>
              <option
                value="Nebraska"
              >
                Nebraska
              </option>
              <option
                value="Nevada"
              >
                Nevada
              </option>
              <option
                value="New Hampshire"
              >
                New Hampshire
              </option>
              <option
                value="New Jersey"
              >
                New Jersey
              </option>
              <option
                value="New Mexico"
              >
                New Mexico
              </option>
              <option
                value="New York"
              >
                New York
              </option>
              <option
                value="North Carolina"
              >
                North Carolina
              </option>
              <option
                value="North Dakota"
              >
                North Dakota
              </option>
              <option
                value="Northern Mariana Islands"
              >
                Northern Mariana Islands
              </option>
              <option
                value="Ohio"
              >
                Ohio
              </option>
              <option
                value="Oklahoma"
              >
                Oklahoma
              </option>
              <option
                value="Oregon"
              >
                Oregon
              </option>
              <option
                value="Palau"
              >
                Palau
              </option>
              <option
                value="Pennsylvania"
              >
                Pennsylvania
              </option>
              <option
                value="Puerto Rico"
              >
                Puerto Rico
              </option>
              <option
                value="Rhode Island"
              >
                Rhode Island
              </option>
              <option
                value="South Carolina"
              >
                South Carolina
              </option>
              <option
                value="South Dakota"
              >
                South Dakota
              </option>
              <option
                value="Tennessee"
              >
                Tennessee
              </option>
              <option
                value="Texas"
              >
                Texas
              </option>
              <option
                value="Utah"
              >
                Utah
              </option>
              <option
                value="Vermont"
              >
                Vermont
              </option>
              <option
                value="Virgin Island"
              >
                Virgin Island
              </option>
              <option
                value="Virginia"
              >
                Virginia
              </option>
              <option
                value="Washington"
              >
                Washington
              </option>
              <option
                value="West Virginia"
              >
                West Virginia
              </option>
              <option
                value="Wisconsin"
              >
                Wisconsin
              </option>
              <option
                value="Wyoming"
              >
                Wyoming
              </option>
            </select>
          </div>
          <input
            fullWidth={true}
            label="Tickets"
            onChange={[Function]}
            sx={
              {
                "marginTop": "20px",
              }
            }
            type="text"
            value=""
          />
        </div>
        <div>
          <button
            className="createGigButton"
            disabled={true}
            onClick={[Function]}
            size="small"
            variant="contained"
          >
            Create
          </button>
          <button
            className="cancelCreateButton"
            onClick={[Function]}
            size="small"
          >
            Cancel
          </button>
        </div>
      </div>
      <div
        className="editGigDialog"
        disableAutoFocus={true}
        disableEnforceFocus={true}
        onClose={[Function]}
        open={false}
      >
        <div>
          Edit Gig
        </div>
        <div
          sx={
            {
              "padding": "10px 10px",
            }
          }
        >
          <div
            sx={
              {
                "marginBottom": "30px",
              }
            }
          >
            Enter all *required
          </div>
          <div
            dateAdapter={[Function]}
          >
            <input
              className="editDateTime"
              label="* Date and Time"
              onChange={[Function]}
              renderInput={[Function]}
              value={null}
            />
          </div>
          <p
            className="venueLabel"
          >
            * Venue
          </p>
          <input
            fullWidth={true}
            label="* City"
            onChange={[Function]}
            sx={
              {
                "marginTop": "20px",
              }
            }
            type="text"
            value=""
          />
          <div
            fullWidth={true}
            sx={
              {
                "marginTop": "20px",
              }
            }
          >
            <p
              id="edit-us-state-label"
            >
              * State
            </p>
            <select
              id="edit-us-state"
              label="* State"
              labelId="edit-us-state-label"
              onChange={[Function]}
              value="Virginia"
            >
              <option
                value="Alabama"
              >
                Alabama
              </option>
              <option
                value="Alaska"
              >
                Alaska
              </option>
              <option
                value="American Samoa"
              >
                American Samoa
              </option>
              <option
                value="Arizona"
              >
                Arizona
              </option>
              <option
                value="Arkansas"
              >
                Arkansas
              </option>
              <option
                value="California"
              >
                California
              </option>
              <option
                value="Colorado"
              >
                Colorado
              </option>
              <option
                value="Connecticut"
              >
                Connecticut
              </option>
              <option
                value="Delaware"
              >
                Delaware
              </option>
              <option
                value="District of Columbia"
              >
                District of Columbia
              </option>
              <option
                value="Federated States of Micronesia"
              >
                Federated States of Micronesia
              </option>
              <option
                value="Florida"
              >
                Florida
              </option>
              <option
                value="Georgia"
              >
                Georgia
              </option>
              <option
                value="Guam"
              >
                Guam
              </option>
              <option
                value="Hawaii"
              >
                Hawaii
              </option>
              <option
                value="Idaho"
              >
                Idaho
              </option>
              <option
                value="Illinois"
              >
                Illinois
              </option>
              <option
                value="Indiana"
              >
                Indiana
              </option>
              <option
                value="Iowa"
              >
                Iowa
              </option>
              <option
                value="Kansas"
              >
                Kansas
              </option>
              <option
                value="Kentucky"
              >
                Kentucky
              </option>
              <option
                value="Louisiana"
              >
                Louisiana
              </option>
              <option
                value="Maine"
              >
                Maine
              </option>
              <option
                value="Marshall Islands"
              >
                Marshall Islands
              </option>
              <option
                value="Maryland"
              >
                Maryland
              </option>
              <option
                value="Massachusetts"
              >
                Massachusetts
              </option>
              <option
                value="Michigan"
              >
                Michigan
              </option>
              <option
                value="Minnesota"
              >
                Minnesota
              </option>
              <option
                value="Mississippi"
              >
                Mississippi
              </option>
              <option
                value="Missouri"
              >
                Missouri
              </option>
              <option
                value="Montana"
              >
                Montana
              </option>
              <option
                value="Nebraska"
              >
                Nebraska
              </option>
              <option
                value="Nevada"
              >
                Nevada
              </option>
              <option
                value="New Hampshire"
              >
                New Hampshire
              </option>
              <option
                value="New Jersey"
              >
                New Jersey
              </option>
              <option
                value="New Mexico"
              >
                New Mexico
              </option>
              <option
                value="New York"
              >
                New York
              </option>
              <option
                value="North Carolina"
              >
                North Carolina
              </option>
              <option
                value="North Dakota"
              >
                North Dakota
              </option>
              <option
                value="Northern Mariana Islands"
              >
                Northern Mariana Islands
              </option>
              <option
                value="Ohio"
              >
                Ohio
              </option>
              <option
                value="Oklahoma"
              >
                Oklahoma
              </option>
              <option
                value="Oregon"
              >
                Oregon
              </option>
              <option
                value="Palau"
              >
                Palau
              </option>
              <option
                value="Pennsylvania"
              >
                Pennsylvania
              </option>
              <option
                value="Puerto Rico"
              >
                Puerto Rico
              </option>
              <option
                value="Rhode Island"
              >
                Rhode Island
              </option>
              <option
                value="South Carolina"
              >
                South Carolina
              </option>
              <option
                value="South Dakota"
              >
                South Dakota
              </option>
              <option
                value="Tennessee"
              >
                Tennessee
              </option>
              <option
                value="Texas"
              >
                Texas
              </option>
              <option
                value="Utah"
              >
                Utah
              </option>
              <option
                value="Vermont"
              >
                Vermont
              </option>
              <option
                value="Virgin Island"
              >
                Virgin Island
              </option>
              <option
                value="Virginia"
              >
                Virginia
              </option>
              <option
                value="Washington"
              >
                Washington
              </option>
              <option
                value="West Virginia"
              >
                West Virginia
              </option>
              <option
                value="Wisconsin"
              >
                Wisconsin
              </option>
              <option
                value="Wyoming"
              >
                Wyoming
              </option>
            </select>
          </div>
          <input
            fullWidth={true}
            label="Tickets"
            onChange={[Function]}
            sx={
              {
                "marginTop": "20px",
              }
            }
            type="text"
            value=""
          />
        </div>
        <div>
          <button
            className="updateGigButton"
            disabled={true}
            onClick={[Function]}
            size="small"
            variant="contained"
          >
            Update
          </button>
          <button
            className="deleteGigButton"
            onClick={[Function]}
            size="small"
            sx={
              {
                "color": "red",
              }
            }
          >
            Delete
          </button>
          <button
            className="cancelEditGigButton"
            onClick={[Function]}
            size="small"
          >
            Cancel
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    style={
      {
        "height": "10px",
      }
    }
  >
    <p>
       
    </p>
  </div>
  <div
    className="elevation3"
    style={
      {
        "margin": "auto",
        "maxWidth": "1000px",
      }
    }
  >
    <section>
      <div
        className="joshBio"
      >
        <div
          style={
            {
              "height": "10px",
            }
          }
        >
          <p>
             
          </p>
        </div>
        <h4
          id="joshbio"
          style={
            {
              "marginBottom": "4px",
              "marginTop": "8px",
            }
          }
        >
          Josh Sherman
        </h4>
        <div>
          <img
            alt="josh01"
            className="alignnone size-full wp-image-49"
            src="/static/imgs/josh01.jpg"
            width="288px"
          />
        </div>
        <p
          className="bioText"
        >
          Josh began playing the trumpet when he was in third grade. He became skilled with the trumpet, continuing with his musical stylings through high school where he was first chair and regular soloist on the marching field. In college Josh studied music and picked up the guitar. He was primarily self-taught on the guitar, although he received help from his many musical relatives, especially his Uncle Mike. Josh went on to found several bands while living in central Florida. He recorded two cds and was played on the local radio stations. Josh also played in many venues including the Sun Cruise casino ship and the Battle of the Bands. His career took a different turn when he moved to Virginia. Leaving his band behind, Josh performed at the annual Bent Mountain pig roast every year since. He joined the College Lutheran Church choir and was a member of the Salem Choral Society. He also performed solo acoustic at local coffee shops. Josh started singing with Maria in the fall of 2011. They fell in love and were married in July of 2012. Vive l’amore!
        </p>
        <blockquote
          style={
            {
              "fontSize": "9pt",
              "fontStyle": "italic",
              "marginTop": "5px",
              "textAlign": "left",
            }
          }
        >
          <p>
            And whenever the harmful spirit from God was upon Saul, David took the lyre and played it with his hand. So Saul was refreshed and was well, and the harmful spirit departed from him.
             
            <strong>
              1 Samuel 16:23
            </strong>
          </p>
        </blockquote>
        <aside
          className="instrumentList"
        >
          <p>
            Instruments that Josh plays:
          </p>
          <ol
            className="instruments"
          >
            <li>
              Lead vocals
            </li>
            <li>
              Harmony vocals
            </li>
            <span>
              <li>
                Acoustic guitar
              </li>
              <li>
                Electric guitar
              </li>
              <li>
                Harmonica
              </li>
              <li>
                Trumpet
              </li>
              <li>
                Kazoo
              </li>
            </span>
          </ol>
        </aside>
      </div>
      <hr />
      <div
        className="mariaBio"
      >
        <h4
          id="mariabio"
          style={
            {
              "marginBottom": "4px",
            }
          }
        >
          Maria Sherman
        </h4>
        <div>
          <img
            alt="maria01"
            className="alignnone size-medium wp-image-50"
            src="/static/imgs/maria01.jpg"
            width="288px"
          />
        </div>
        <p
          className="bioText"
        >
          Maria started her singing career at the age of 4 when she performed at the JaMar Rec Center in St. Petersburg, Florida. Maria continued adding to her musical repertoire by learning piano, alto saxophone, tenor saxophone, bassoon, and marching tenors. She earned a minor in voice at Roanoke College and did a variety of chorus, musical theatre, and solo performances while teaching in the Roanoke County Schools. Although classically trained, Maria loves singing rock and Christian music with Josh. She is currently playing bass guitar and plays accordian. Josh is the most wonderful husband and is the driving force for the couple; Maria is a fabulous wife and is the organization behind the duo.
        </p>
        <blockquote
          style={
            {
              "fontSize": "9pt",
              "fontStyle": "italic",
              "marginTop": "5px",
              "textAlign": "left",
            }
          }
        >
          <p>
            Whoever sings songs to a heavy heart is like one who takes off a garment on a cold day, and like vinegar on soda.
             
            <strong>
              Proverbs 25:20
            </strong>
          </p>
        </blockquote>
        <aside
          className="instrumentList"
        >
          <p>
            Instruments that Maria plays:
          </p>
          <ol
            className="instruments"
          >
            <li>
              Lead vocals
            </li>
            <li>
              Harmony vocals
            </li>
            <span>
              <li>
                Bass guitar
              </li>
              <li>
                Accordion
              </li>
              <li>
                Bassoon
              </li>
              <li>
                Saxophone
              </li>
              <li>
                Tri-tom
              </li>
            </span>
          </ol>
        </aside>
      </div>
    </section>
    <p>
       
    </p>
    <p>
       
    </p>
  </div>
</div>
`;
